libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -81,6 +80,11 @@ bool SyntaxChecker::visit(PragmaDirective const& _pragma)
 			);
 		m_versionPragmaFound = true;
 	}
+	else if (literal == "checkAssertionsZ3")
+	{
```

Comment:
Do you plan to add something here? If not wouldn't `if (literal != ...) syntaxError` be enough?